tools/quantize/requirements.txt

```diff
@@ -1,4 +1,5 @@
 opencv-python>=4.5.4.58
 onnx
 onnxruntime
+onnxruntime-extensions
```

Comment:
Why do we need this package?

Reply:
OK

Reply:
Because there seems to be some problems with the old version, perhaps they have been solved now. I will check this again

Reply:
I have met the same problem as https://github.com/intel/neural-compressor/issues/94 when I only install onnxruntime. But after I install the `onnxruntime-extensions` mentioned by the maintainers from the link, the problem can be solved and jump to the next error message. Just let me check it again. I think it's my fault.

Reply:
I have checked this problem. The `neural-compressor` did need the `onnxruntime-extensions`, but it will not install automatically just like `onnxruntime`.

![image](https://user-images.githubusercontent.com/49567307/196117829-f943e9b6-5785-4e3d-bb30-a6e4650d08b1.png)

Reply:
OK, we can keep this change. Please update for the other comment, and we can merge this pr.